dashboard/dashboard_builder.py

```diff
@@ -356,43 +367,60 @@ def main():
     checker = compatibility_checker.CompatibilityChecker()
     store = compatibility_store.CompatibilityStore()
 
-    packages = [
-        package.Package(install_name) for install_name in args.packages]
-    logging.info("Getting self compatibility results...")
-    package_to_results = store.get_self_compatibilities(packages)
-    logging.info("Getting pairwise compatibility results...")
-    pairwise_to_results = store.get_compatibility_combinations(packages)
-
-    package_with_dependency_info = {}
-    for pkg in configs.PKG_LIST:
-        dep_info = store.get_dependency_info(pkg)
-        package_with_dependency_info[pkg] = dep_info
-
-    results = _ResultHolder(
-        package_to_results,
-        pairwise_to_results,
-        package_with_dependency_info,
-        checker,
-        store)
-
-    dashboard_builder = DashboardBuilder(packages, results)
-
-    # Build the pairwise grid dashboard
-    logging.info('Starting build the grid...')
-    grid_html = dashboard_builder.build_dashboard(
-        'dashboard/grid-template.html')
-    grid_path = os.path.dirname(os.path.abspath(__file__)) + '/grid.html'
-    with open(grid_path, 'wt') as f:
-        f.write(grid_html)
-
-    # Build the dashboard main page
-    logging.info('Starting build the main dashboard...')
-    main_html = dashboard_builder.build_dashboard(
-        'dashboard/main-template.html')
-
-    main_path = os.path.dirname(os.path.abspath(__file__)) + '/index.html'
-    with open(main_path, 'wt') as f:
-        f.write(main_html)
+    instance_flag = '-instances={}=tcp:{}'.format(
+        INSTANCE_CONNECTION_NAME, PORT)
+    cloud_sql_proxy_path = './cloud_sql_proxy'
+
+    try:
+        # Run cloud_sql_proxy
+        process = popen_spawn.PopenSpawn([cloud_sql_proxy_path, instance_flag])
+        process.expect('Ready for new connection', timeout=5)
+
+        packages = [
+            package.Package(install_name) for install_name in args.packages]
+        logging.info("Getting self compatibility results...")
```

Comment:
Be consistent when using quotation marks.

Reply:
Done.